src/api/cpp/cvc5.cpp

```diff
@@ -4862,6 +4864,8 @@ Term Solver::mkBVFromStrHelper(uint32_t size,
         << "Overflow in bitvector construction (specified bitvector size "
         << size << " too small to hold value " << s << ")";
   }
+  // Doesn't happen automatically.
+  val = val.modByPow2(size);
```

Comment:
This is done in the `BitVector` constructor, actually: https://github.com/cvc5/cvc5/blob/master/src/util/bitvector.h#L33

Reply:
Good point. Removed.